src/pybind/mgr/dashboard_v2/models/cluster.py

```diff
@@ -1,9 +1,18 @@
+# -*- coding: utf-8 -*-
+
+import json
+import logging
+
+from .nodb import NodbModel, CharField, JsonField, ValidationError, \
+                  bulk_attribute_setter
 from ..models.send_command_api import SendCommandApiMixin
 from ..tools import ExternalCommandError
-from .nodb import *
 from ..tools import RESTController  # pylint: disable=W0611
 
 
+logger = logging.getLogger(__name__)
```

Comment:
Afaik we should not use the python logger, but the logger exposed in the module. That's why I didn't add it in the first place.

Reply:
I agree. The problem is that in `nodb.py` was added a python logger, https://github.com/openattic/ceph/commit/7f1729c224afccf7870d81ace01c04ba31183807#diff-a80a3792a3b6f9c36a06ae4bf915ed70R18 , that was being imported in this file.
I added that line for now just to make sure that the code runs, but I'm expecting that someone will open a PR to change this logger for the manager module's logger.